core/src/main/clojure/xtdb/information_schema.clj

```diff
@@ -199,6 +216,28 @@
      :nspowner nspowner
      :nspacl nil}))
 
+(def pg-ams [{:oid 2
+              :amname "heap"
+              :amhandler "heap_tableam_handler"
+              :amtype "t"
+             }
+             ;; FIXME this should prolly be something else (arrow-HAMTrie-handler?)
```

Comment:
this is the sort of comment to either resolve beforehand, or explicitly decide not to handle now